.github/workflows/ci_pipeline.yml

```diff
@@ -107,14 +103,38 @@ jobs:
           verbose: true
         if: ${{ matrix.upload_artifacts }}
 
-      - name: Static analysis
-        working-directory: ${{ github.workspace }}
-        run: ./src/beluga/tools/run-clang-tidy.sh
+  build-docs:
+    needs: build-test
+    runs-on: ubuntu-22.04
+    steps:
+      - name: Checkout repository
+        uses: actions/checkout@v3
+
+      - name: Setup python
+        uses: actions/setup-python@v4
+        with:
+          cache: pip
+
+      - name: Install tooling dependencies
+        run: >-
+          sudo apt-get update &&
+          sudo apt-get install -y doxygen git make &&
```

Comment:
We might be missing a package here according to doc dependencies: https://github.com/Ekumen-OS/beluga/blob/main/beluga/package.xml#L31C15-L31C33

Maybe `texlive-latex-base` and `graphviz`?

```
error opening bib2759.bbl: No such file or directory
Preprobibtex command failed: No such file or directory
error: Problems running bibtex. Verify that the command 'perl --version' works from the command line. Exit code: 2
```

Reply:
Indeed. I may add a new Docker image in a separate PR. `texlive` is large.

Reply:
Alright, `texlive-latex-base` seems small enough for now. I'll defer more infra work to https://github.com/Ekumen-OS/beluga/issues/354.